openpype/settings/defaults/project_settings/fusion.json

```diff
@@ -1,5 +1,13 @@
 {
     "imageio": {
+        "ocio_config": {
```

Comment:
Isn't this duplicate of the `"ocio"` key here?

Reply:
I had implemented the new way into Nuke so inspiration of how this works could be taken from [here](https://github.com/ynput/OpenPype/pull/4195/files#diff-d496b587c94a631576b2d043b85b9c1b95394f41641e4681339c49d7a493176dR1888). Next step on the roadmap will be integrated in hosts.